tests/integration/test_windows_fake_fs_token.py

```diff
@@ -0,0 +1,142 @@
+import random
+import pytest
+from typing import List
+
+from canarytokens.models import (
+    WindowsFakeFSTokenHistory,
+    WindowsFakeFSTokenRequest,
+    WindowsFakeFSTokenResponse,
+)
+from canarytokens.webhook_formatting import TokenAlertDetailGeneric
+
+from tests.utils import (
+    clear_stats_on_webhook,
+    trigger_windows_fake_fs_token,
+    create_token,
+    get_stats_from_webhook,
+    get_token_history,
+    run_or_skip,
+    v3,
+)
+
+
+@pytest.mark.parametrize("version", [v3])
+@pytest.mark.parametrize(
+    "file_name, process_name",
+    [
+        ("doc b.docx", "explorer.exe"),
+        ("doc b.doc", "explorer.exe"),
+    ],
+)
+def test_windows_fake_fs_token_fires(
+    file_name: str,
+    process_name: str,
+    webhook_receiver,
+    version,
+    runv2,
+    runv3,
+):
+    """
+    Tests the Windows Fake FS token.
+    """
+    run_or_skip(version, runv2=runv2, runv3=runv3)
```

Comment:
Do we still need this (and the parametrize on line 23)?

Reply:
removed